### DISCUSSION TITLE
Copy Location widgets when copy pages to other languages

### DISCUSSION THREAD

**Discussion by Katrienvh** [forkcms/forkcms#3341] [Ideas / Feature Requests]
## Type

- Enhancement

## Problem description

if you run /pages/copy?from=en&to=nl for example, it will copy alle the pages with content, content blocks, ... 
It should be very nice if the location widgets would also be copied. I think most of the time u use the location widget the location won't change when the language of the location changes.

**Comment by jeroendesloovere** [comment]
I will wait until <s>#2349 (Location entity)</s> and #2422 (copy modules) are merged to fix this one.
